src/sentry/integrations/slack/integration.py

```diff
@@ -283,16 +305,25 @@ def _request_channel_info(pipeline):
             # TODO(meredith): subclass the ApiError and make a SlackApiError so we can
             # reraise the other errors
         except ApiError as e:
+            logger.info(
+                "slack.request_channel_info.response-error",
+                extra={
+                    "error": six.text_type(e),
+                    "channel": channel["id"],
+                    "integration_id": integration.id,
+                },
+            )
             # adds the channel to our dict grouped by the error message which could
```

Comment:
This comment is now outdated so might wanna change it

```python 
any channels that we couldn't verify are added under `"not_found"`, to be later surface in
email to user in `post_migration.py`
```

something like that i dk